### DISCUSSION TITLE
User and password unknown in Elastic

### DISCUSSION THREAD

**Discussion by canix1** [Security-Onion-Solutions/securityonion#11389] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

2

### RAM

8GB

### Storage for /

240 GB

### Storage for /nsm

240 GB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I can login to the SOC console but when I click Kibana I'm unable to login. I get "Username or password is incorrect".
I have make sure I have changed my password at least once. I have tested to create a new users too.

I have tried the command: sudo FORCE_SYNC=true so-user sync.

This command " sudo salt-call pillar.get elasticsearch:auth:enabled" gives me:
local:

Result from sudo salt-call state.highstate:
local:
    Data failed to compile:
----------
    The function "state.highstate" is running as PID 1387179 and was started at 2023, Sep 26 11:24:05.242796 with jid 20230926112405242796

Log:  /opt/so/log/elasticsearch/securityonion.log 
[2023-09-26T10:45:02,857][WARN ][org.elasticsearch.xpack.security.authc.RealmsAuthenticator] Authentication to realm default_file failed - Password authentication failed for r****@***.com

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment, marked as answer]
8GB RAM does not meet the minimum requirement for a standalone installation:
https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments

Insufficient RAM can cause all kinds of problems.

**Reply by canix1** [reply]
That could be it, this time I installed the Desktop version so it's probably eating up the memory. I will re-install without the Desktop.
